docs/static/arcsight-module.asciidoc

```diff
@@ -265,6 +267,123 @@ servers. (You may want more than one in case a server is down.)
 +
 A list of Event Broker topics to subscribe to.
 
+*`var.input.eventbroker.security_protocol`*::
++
+--
+* Value can be any of: `PLAINTEXT`, `SSL`, `SASL_PLAINTEXT`, `SASL_SSL`
+* Default value is `"PLAINTEXT"`
+--
+Security protocol to use, which can be either of PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL. If you specify anything other than PLAINTEXT then you need to also specify some of the options listed below. When specifying `SSL` or `SASL_SSL` you should supply values for the options prefixed with `ssl_`, when specifying `SASL_PLAINTEXT` or `SASL_SSL` you should supply values for `jaas_path`, `kerberos_config`, `sasl_mechanism` and `sasl_kerberos_service_name`.
+
+*`var.input.eventbroker.ssl_key_password`*::
++
+--
+* Value type is <<password,password>>
```

Comment:
<<password,password> hasn't been defined in https://github.com/elastic/docs-tools/blob/master/versioned_plugins.rb, so links aren't resolving (lines 281, 297, 321, 432)

@guyboertje I can put up a PR for this if you think we should link rather than hard code. Thinking back to our discussion in https://github.com/elastic/docs-tools/pull/29 for precedent. Please let me know how you'd like to proceed.

Reply:
UPDATE:  Target text is already in https://www.elastic.co/guide/en/logstash/6.4/configuration-file-structure.html, so question posed above is not relevant.  Prob no reason not to add cleanup item.